Modules/signalmodule.c

```diff
@@ -504,6 +504,75 @@ signal_getsignal_impl(PyObject *module, int signalnum)
     }
 }
 
+
+/*[clinic input]
+signal.strsignal
+
+    signalnum: int
+    /
+
+Return the system description of the given signal.
+
+The return values can be such as 'Interrupt', 'Segmentation fault', etc.
+Returns None if the signal is not recognized.
+[clinic start generated code]*/
+
+static PyObject *
+signal_strsignal_impl(PyObject *module, int signalnum)
+/*[clinic end generated code: output=44e12e1e3b666261 input=5369187c5b204b2a]*/
+{
+    int position;
+    const char *res, *pch;
```

Comment:
According to [POSIX](http://pubs.opengroup.org/onlinepubs/9699919799/functions/strsignal.html), `strsignal` returns non-const `char*`, so let's keep it like that.

Reply:
I don't understand this one. Why would we want to keep it not const? We're not planning on modifying it, adding constness to make sure of that doesn't hurt.

Reply:
`Py_BuildValue` doesn't specify that it takes only `const` arguments, so you may get compiler warnings or even errors depending on compiler settings. The benefit of using `const` in such a simple function seem rather low, so it's better to drop it iMHO.

Reply:
Got it, I just looked at the docs of Py_Buildvalue. I fixed it, thanks.